keyboards/angel64/config.h

```diff
@@ -48,16 +48,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // #define BACKLIGHT_BREATHING
 // #define BACKLIGHT_LEVELS 3
 
-// #define RGB_DI_PIN E2
-// #ifdef RGB_DI_PIN
-//   #define RGBLED_NUM 64
-//   #define RGBLIGHT_HUE_STEP 8
-//   #define RGBLIGHT_SAT_STEP 8
-//   #define RGBLIGHT_VAL_STEP 8
-//   #define RGBLIGHT_LIMIT_VAL 255 /* The maximum brightness level */
-//   #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
-// /*== all animations enable ==*/
-//   #define RGBLIGHT_ANIMATIONS
+#define RGB_DI_PIN D3
+#ifdef RGB_DI_PIN
+   #define RGBLED_NUM 64
+   #define RGBLIGHT_HUE_STEP 8
+   #define RGBLIGHT_SAT_STEP 8
+   #define RGBLIGHT_VAL_STEP 8
+   #define RGBLIGHT_LIMIT_VAL 25 /* The maximum brightness level */
```

Comment:
Is it intentional to limit the value so low?

Reply:
Yeah, that's super low.  That's 10% of the maximum, and may not be easily noticeable in many cases.

Reply:
Intentionally lowered.
It does not have enough power to light the 64 RGBLIGHTs.

USB 2.0 provides up to 500mA of power.
SK6812mini uses up to 60mA of power.

500(mA)/ 64(LEDs) / 60(mA) *100 = 13.02% limit.

Reply:
60mA is for pure white, it avgs out to 20mA for the individual rgb. Also the power draw is not linear, especially after the brightness is applied to make it appear linear. As an example, the Sol r2 uses 128 of the same LEDs and at 104 brightness, all but 3 effects are under 500mA draw. Turning off a chunk of the LEDs down to 70 and 120mA will bring nearly all but pure white to under 500mA draw. So 25 is really way too low. I'd recommend 120-128

Reply:
Also just because the spec says 500mA max, doesn't mean your pc or hub won't easily allow more. Mine allows me to hit full 255 brightness pulling nearly a full 3A.

Reply:
I confirmed that it would not work if it was 255.
I set the value to 100.
It may work with a slightly higher value.
I do not want to set a higher value, as the PC may be destroyed.

Reply:
@XScorpion2 that may be what the spec says, but realistically, if the controller is wired correctly, and it tries to use more than that, it will crash the controller. 

I've seen this happen with my corne keyboard.